packages/teleport/src/services/mfa/types.ts

```diff
@@ -0,0 +1,13 @@
+export type MfaDevice = {
+  id: string;
+  name: string;
+  typeText: 'Hardware Key' | 'Authenticator App';
```

Comment:
I am a bit confused with this `typeText`. Why does it need to be `typed` ? If this is a description then `description: string` should be enough. 

```
  id: string;
  name: string;
  type:  'TOTP' | 'U2F' ...,
  description: string;
```

Reply:
@alex-kovoy In the table we display either "Hardware Key" or "Authenticator App"

![image](https://user-images.githubusercontent.com/56373201/129997394-6b21ffcd-a679-4f63-802a-0bf5dd12a6d0.png)

The alternative is using `type: 'TOTP' | 'U2f'` and then creating something like a `DeviceTypeCell` that checks the `type` and then for example renders `"Hardware Key"` if `type === 'U2F`, but we won't use the strings `'TOTP'` or `'U2F'` anywhere else, so I am just doing that check on the service layer and returning the device object with the formatted text already there.